models/shopify_graphql.yml

```diff
@@ -0,0 +1,1202 @@
+      - name: inventory_item_updated_at
+        description: The date and time (ISO 8601 format) when the inventory item was last modified.
+      - name: is_location_deleted
+        description: "{{ doc('_fivetran_deleted') }}"
+      - name: is_location_active
+        description: >
+          Boolean representing whether the location is active. If true, then the location can be used to sell products, stock inventory, and fulfill orders.
+      - name: address_1
+        description: The location's street address.
+      - name: address_2
+        description: The optional second line of the location's street address.
+      - name: city
+        description: The city the location is in.
+      - name: country
+        description: The country the location is in (two-letter code).
+      - name: country_code
+        description: The two-letter code (ISO 3166-1 alpha-2 format) corresponding to country the location is in.
+      - name: location_created_at
+        description: The date and time (ISO 8601 format) when the location was created.
+      - name: is_legacy_location
+        description: >
+          Boolean representing whether this is a fulfillment service location. If true, then the location is a fulfillment service location. 
+          If false, then the location was created by the merchant and isn't tied to a fulfillment service.
+      - name: location_name
+        description: The name of the location.
+      - name: phone
+        description: The phone number of the location. This value can contain special characters, such as - or +.
+      - name: province
+        description: The province, state, or district of the location.
+      - name: province_code
+        description: The province, state, or district code (ISO 3166-2 alpha-2 format) of the location.
+      - name: location_updated_at
+        description: The date and time (ISO 8601 format) when the location was last updated.
+      - name: zip
+        description: The zip or postal code.
+      - name: variant_barcode
+        description: The barcode, UPC, or ISBN number for the product.
+      - name: variant_created_at
+        description: The date and time (ISO 8601 format) when the product variant was created.
+      - name: variant_id
+        description: The unique numeric identifier for the product variant.
+      - name: variant_inventory_policy
+        description: Whether customers are allowed to place an order for the product variant when it's out of stock.
+      - name: variant_inventory_quantity
+        description: An aggregate of inventory across all locations. To adjust inventory at a specific location, use the InventoryLevel resource.
+      - name: variant_price
+        description: The price of the product variant in shop currency.
+      - name: variant_media_id
+        description: The unique numeric identifier for a product variant's media. 
```

Comment:
This field is not included in the end model. Is that expected?

Reply:
Should be included if `shopify_gql_using_product_variant_media` is set to True